LobbyManager.h

```diff
@@ -0,0 +1,60 @@
+#ifndef TICTACTOE_SERVER_LOBBYMANAGER_H
+#define TICTACTOE_SERVER_LOBBYMANAGER_H
+
+#include "SearchManager.h"
+#include <unordered_map>
+#include <random>
+
+class LobbyManager {
+public:
+	struct Lobby {
+		SearchManager::PlayerInfo player1;
+		SearchManager::PlayerInfo player2;
+		bool player1_active;
+		bool player2_active;
+		unsigned search_id;
+
+		Lobby(SearchManager::PlayerInfo player1, SearchManager::PlayerInfo player2, unsigned id) : player1(player1), player2(player2), search_id(id){}
+
+	};
+
+private:
+	unordered_map<unsigned, Lobby> active_lobbies;
+
+public:
+
+	void create_lobby(SearchManager::PlayerInfo player1, SearchManager::PlayerInfo player2, unsigned id) {
+		active_lobby[std::mt19937(std::random_device()())()] = Lobby(player1, player2, id);
+	}
+
+	bool check_for_active_lobby_with_id(std::string player_id ) {
```

Comment:
The name of this method doesn't match it's functionality & input parameters. The name suggests that we pass the lobby id, whereas we are passing a player id. Rename or refactor.